apps/studio/components/interfaces/Settings/Addons/IPv4SidePanel.tsx

```diff
@@ -92,9 +95,17 @@ const IPv4SidePanel = () => {
       onCancel={closePanel}
       onConfirm={onConfirm}
       loading={isLoading || isSubmitting}
```

Comment:
I think this needs to be `loading={isLoading || isLoadingEntitlement || isSubmitting}` ?